src/MSBuild/MSBuild.csproj

```diff
@@ -7,6 +7,7 @@
     <OutputType>Exe</OutputType>
     <TargetFrameworks>$(RuntimeOutputTargetFrameworks)</TargetFrameworks>
     <PlatformTarget>$(RuntimeOutputPlatformTarget)</PlatformTarget>
+    <UseAppHost>false</UseAppHost>
```

Comment:
Comment why please

Reply:
This suggestion justifies good commit messages, I'm thanking my past self right now 😁